package.json

```diff
@@ -31,18 +31,18 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.4.3",
     "@babel/preset-typescript": "^7.3.3",
-    "@callstack/eslint-config": "^4.2.0",
-    "@types/jest": "^24.0.11",
-    "@types/webpack": "^4.4.29",
+    "@callstack/eslint-config": "^7.0.0",
+    "@types/jest": "^24.0.17",
     "@types/mkdirp": "^0.5.2",
-    "@types/node": "^11.13.4",
+    "@types/node": "^12.7.0",
```

Comment:
FYI according to @Esemesek if we target Node 10, we should be using `@types/node@^10.0.0` to make sure we use compatible APIs